src/test/java/com/ixortalk/organization/api/graphql/GraphQL_Organization_IntegrationAndRestDocTest.java

```diff
@@ -0,0 +1,141 @@
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.ixortalk.organization.api.graphql;
+
+import com.ixortalk.organization.api.AbstractSpringIntegrationTest;
+import com.ixortalk.organization.api.config.TestConstants;
+import com.ixortalk.organization.api.util.GraphQLUtil;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.config.EncoderConfig.encoderConfig;
+import static io.restassured.http.ContentType.JSON;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.restdocs.headers.HeaderDocumentation.requestHeaders;
+import static org.springframework.restdocs.operation.preprocess.Preprocessors.*;
+import static org.springframework.restdocs.restassured3.RestAssuredRestDocumentation.document;
+
+public class GraphQL_Organization_IntegrationAndRestDocTest extends AbstractSpringIntegrationTest {
+
+    @Test
+    public void asAdmin() {
+        String query =
+                "{" +
+                        "  organizationsPage(page:0, size:2, sort:\"name\", direction:\"asc\", filter:\"name:org\") {" +
+                        "content {" +
+                    "          id" +
+                    "          name" +
+                    "          phoneNumber" +
+                    "          emailAddress" +
+                    "          image" +
+                    "          logo" +
+                    "          roles {" +
+                    "              id" +
+                    "              name" +
+                    "          }" +
+                    "          users {" +
+                    "              id" +
+                    "              login" +
+                    "          }" +
+                    "          address {" +
+                    "              streetAndNumber" +
+                    "              postalCode" +
+                    "              city" +
+                    "              country" +
+                    "          }" +
+                    "      }" +
+                        GraphQLUtil.PAGE_FIELDS +
+                        "  }" +
+                        "}";
+
+        JsonPath jsonPath = given()
+                .config(RestAssured.config().encoderConfig(encoderConfig().encodeContentTypeAs("application/graphql", ContentType.TEXT)))
```

Comment:
Can we configure this top level once instead of repeating it for all tests?

Reply:
I was referring only to the `RestAssured.config` part, it gets configured globally here: `com.ixortalk.organization.api.AbstractSpringIntegrationTest#setupRestAssuredAndRestDocSpec` for all tests, I suppose this specific thing can go there too?

The chaining `given().auth()...body()...` can remain in specific tests.

Reply:
@wjans Will move the config part... 
I prefer to leave the utility method for the sake fo DRYness.